data/sample_cities/ZZZ_template_city/run_b_CTCM_processing.sh

```diff
@@ -50,14 +57,17 @@ minutes=$(((runtime % 3600) / 60))
 echo "Total runtime: ${hours} hour(s) and ${minutes} minute(s)"
 echo
 
-# Send Slack notification (if webhook URL is set)
-slack_mention="<@U031HB1JNRZ>"  # Replace with the Slack user. Default is Chris
+# # Send Slack notification (if webhook URL is set)
+# slack_mention="<@U031HB1JNRZ>"  # Replace with the Slack user. Default is Chris
 
-message="$slack_mention CTCM processing for $city_folder completed in ${hours} hour(s) and ${minutes} minute(s)."
-curl -X POST -H 'Content-type: application/json' --data "{\"text\": \"$message\"}" https://hooks.slack.com/services/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
+# message="$slack_mention CTCM processing for $city_folder completed in ${hours} hour(s) and ${minutes} minute(s)."
+# curl -X POST -H 'Content-type: application/json' --data "{\"text\": \"$message\"}" https://hooks.slack.com/services/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
 
 # Shut down ec2 instance (uncomment the next line to enable)
```

Comment:
I guess we can remove the "(uncomment the next line to enable)" part of the comment